chromium_src/chrome/browser/sharing_hub/sharing_hub_model.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/sharing_hub/sharing_hub_model.h"
+#include "brave/app/brave_command_ids.h"
+#include "brave/grit/brave_generated_resources.h"
+#include "chrome/grit/generated_resources.h"
+
+int GetOriginalSharingHubCopyLinkID() {
```

Comment:
should be static or in anonymous ns.
also can be replaced with `static auto IDS_SHARING_HUB_COPY_LINK_LABEL_ORIGINAL = IDS_SHARING_HUB_COPY_LINK_LABEL;`

Reply:
done, thanks